### DISCUSSION TITLE
Feature Request - Backup without zipping and back up only files that have changed

### DISCUSSION THREAD

**Discussion by Sas-KP** [MikeMaximus/gbm#301] [Ideas]
I have been searching for a long time for a software like this and this is ALMOST perfect.

I want a software that automatically backs up my games after I'm done playing, like how Steam Cloud does, but without the additional zipping of the files.

Some game have a bunch of saves, like Baldur's Gate 3 (over 500MB), and zipping up all those files every time wears down your SSD.

Also, it would be nice if it would only back up the files that have changed. For example, I have 15 quick-saves and 15 auto-saves. After a game session most of them are un-changed (some of the old ones get deleted when new ones get created but the ones in-between stay the same).

**Comment by MikeMaximus** [comment]
Thanks for the suggestions Sas-KP.

> I want a software that automatically backs up my games after I'm done playing, like how Steam Cloud does, but without the additional zipping of the files.  Some game have a bunch of saves, like Baldur's Gate 3 (over 500MB), and zipping up all those files every time wears down your SSD.

I won't say never, but I'm unlikely to add the ability to backup files without using compression.

If you're concerned about SSD wear, the best thing you can do is change the "Backup Folder" and "Temp Folder" in [Settings -> Files and Folders](https://mikemaximus.github.io/gbm-web/manual.html#settings_filesandfolders) to another location, like an HDD.

> Also, it would be nice if it would only back up the files that have changed. For example, I have 15 quick-saves and 15 auto-saves. After a game session most of them are un-changed (some of the old ones get deleted when new ones get created but the ones in-between stay the same).

You can use [Differential Backups](https://mikemaximus.github.io/gbm-web/manual.html#differential) to only backup files that have been added or changed after each session.
